docs/examples/tests/responses/test_background_tasks.py

```diff
@@ -11,6 +13,9 @@
     from _pytest.logging import LogCaptureFixture
 
 
+pytestmark = pytest.mark.usefixtures("reset_httpx_logging")
```

Comment:
What does this do?

Reply:
It's the same as `@pytest.mark.usefixtures()` just on a module level, so the fixture does not need to be added to each test case individually.

Reply:
Oh, handy!